main.tex

```diff
@@ -169,96 +169,102 @@ \subsection{Phases}
     \item During Signing Phase, inputs those do not sign are non-compliant inputs and they shall be issued a penalties.
 \end{enumerate}
 
-The point of the cryptography in WabiSabi is to ensure it does not worth for any non-compliant input to sign the final CoinJoin transaction, otherwise non-compliant inputs could not be identified. We achieve this by ensuring no user can register outputs those sum is larger than its registered inputs' sum.
+The point of the cryptography in WabiSabi is to ensure that honest participants would always agree to sign the final CoinJoin transaction because their requested outputs were accepted by the coordinator. Anonymous credentials allow the coordinator to verify that amounts of each user's output registrations are funded by input registrations without learning specific relationships between inputs and outputs.
```

Comment:
```suggestion
The point of the cryptography in WabiSabi is to ensure that honest participants would always agree to sign the final CoinJoin transaction because their requested outputs were accepted by the coordinator. Anonymous credentials allow the coordinator to verify that amounts of each user's output registrations are funded at input registrations without learning specific relationships between inputs and outputs.
```